### ISSUE TITLE
Feature: Don't disturb all tabs except one

### ISSUE THREAD

**Issue by errotu** [saenzramiro/rambox#829]
First: Thanks a lot for this great application!

I love using Rambox and actually united all my messenger services in your tool. As I am using Rambox also quite a lot for Skype calls, a "Don't disturb all tabs except this" - functionality would be great. Right now, when I'm having a Skype call, I still hear all the notifications from the other messenger services. But when I use the "Don't disturb"-functionality, it also disables the sound in the skype call.

Once again, thanks a lot for this great tool!

**Comment by MustafaHaddara** [comment]
Similarly, it would be nice to have a per-service Do Not Disturb functionality, so we can selectively toggle notifications. 

Right now I can achieve this by adjusting the settings for the service, but it would be useful to be able to do that from the main screen instead of having to open the Service Settings modal window.

**Comment by saenzramiro** [comment]
Right, I agree with you guys. I will think about this in a future release.
